### ISSUE TITLE
BioModel Discussion Issue for eos5cl7

### ISSUE THREAD

**Issue by Zainab-ik** [ersilia-os/eos5cl7#9]
**Summary !!!**

- **BioModel Name**: Pereira2020 -  Discovering growth Inhibitors of Neisseria gonorrhoeae with Machine Learning

- **BioModel Tag**: fair AIML, Machine learning model, Ersilia

- **Metadata Comments**: 
1. The invitro experiment was to validate the growth inhibition activity of the 20 compounds predicted by the bayesian model, that counts as the experimental validation of the model, right? what about the synthetic chemistry experiment?
2. There was a mention of mouse liver microsome (MLM) stability and kinetic aqueous solubility (S), these won't be pertaining to the models, right?

- **Contributor**:
> 
 | roles             | Names | 
 | :---------------- | :------: | 
 | Curator        |   @Zainab-ik   | 
 | Code contributor      |   @Richiio        | 
 | Reviewer       |   @GemmaTuron    | 

- **Curation status** :  manually curated

- **Annotation file**: [here](https://docs.google.com/spreadsheets/d/1ycJ5t7WOB05prSRJ4ZKNYNv6vrugzTps2d3m5UJhXyU/edit?usp=drive_link)

**Comment by GemmaTuron** [comment]
The invitro experiment was to validate the growth inhibition activity of the 20 compounds predicted by the bayesian model, that counts as the experimental validation of the model, right? what about the synthetic chemistry experiment? The in vitro experiment is the validation indeed. The synthetic chemistry is just how they synthesise those compounds for testing

There was a mention of mouse liver microsome (MLM) stability and kinetic aqueous solubility (S), these won't be pertaining to the models, right? indeed, they are not

**Comment by GemmaTuron** [comment]
There is in vtiro validation of the results, so shouldn't we mention it?
Why is QSAR not added here?

**Comment by Zainab-ik** [comment]
> There is in vtiro validation of the results, so shouldn't we mention it? Why is QSAR not added here?

Yes, It's mentioned as experimental validation and in-vitro model. 
QSAR added.

**Comment by GemmaTuron** [comment]
Hi @Zainab-ik 
As we just discussed, the in vitro model should be referring only to the validation, note this with the DOME annotation. The hits field does not add any information
The metrics should be revised
